libbables.js

```diff
@@ -1,78 +0,0 @@
-var libs, repeatables, repeats, form, fields, submitEnabled, submitButton;
-
-function verifySubmitReady() {
-//    if (submitEnabled && this.value === "") {
-//        submitEnabled = false
-//    } else {
-//        submitEnabled = true;
-//        var counter = 0;
-//        while (submitEnabled && counter < fields.length) {
-//            submitEnabled = submitEnabled && (fields[counter].value != "")
-//        }
-//    }
-//    submitButton.disabled = !submitEnabled
-}
-
-function createFields() {
-    fields = [];
-    form = document.createElement("div");
-//    var br = document.createElement("br");
-    var label, input, labelInputGroup;
-    for (var k = 0; k < libs.length; k++) {
-        labelInputGroup = document.createElement("div");
-        labelInputGroup.className = "labelInputGroup";
-        label = document.createElement("span");
-        label.className = "label";
-        label.innerHTML = libs[k].innerHTML;
-        labelInputGroup.appendChild(label);
-        input = document.createElement("input");
-        input.addEventListener("change", verifySubmitReady);
```

Comment:
This can actually be simplified by listening to the entire form rather than each individual control. I've updated this in the class version.